SampleProject/VOKCoreDataManagerTests/VOKManagedObjectAdditions.m

```diff
@@ -156,17 +154,17 @@ - (void)testRecordInsertionBackgroundThreadConvenienceReturningManagedObjectsArr
                                  @"numberOfHats" : @1,
                                  },
                              ];
-
+    
     [VOKThing vok_addWithArrayInBackground:importArray
-                               completion:^(NSArray *arrayOfManagedObjects) {
```

Comment:
I don't think you can use `instancetype` as an Obj-C generic type.  As it is now, that method returns `NSArray<__kindof NSManagedObject *> *` in Obj-C.  

With the Swift additions, you **can** get a properly typed array of your managed object subclass either through the generic [`importArray`](https://github.com/vokal/Vokoder/blob/master/Pod/Classes/Swift/VOKCoreDataManager%2BSwift.swift#L37-L46) function, or by adopting the `VokoderTypedManagedObject` protocol and using the [`vok_import`](https://github.com/vokal/Vokoder/blob/master/Pod/Classes/Swift/VokoderTypedManagedObject.swift#L14-L33) function.